### ISSUE TITLE
model does not support gradient calculation if no_grad = False?

### ISSUE THREAD

**Issue by hedongyan** [automl/TabPFN#69]
I use the TabPFNClassifier and set the no_grad=False. However, the model can not return a tensor with gradient. What should I do to get the gradient of the model output and use loss.backward to get the gradient of the input?

**Comment by DhanshreeA** [comment]
Do your input tensors have require_grad set to True?
